requirements.txt

```diff
@@ -1,5 +1,5 @@
 autotest>=0.16.2; python_version < '3.0'
-aexpect>=1.6.3;python_version >= '3.0'
+aexpect==1.6.2;python_version >= '3.0'
```

Comment:
this version:1.6.3 is intended for fixing kamil's job cases

Reply:
yes, but it blocks the whole v2v testing. That's not expected. v2v testing is more important that the new job cases, isn't it?